### DISCUSSION TITLE
Welcome to Warga Discussions!

### DISCUSSION THREAD

**Discussion by gimid** [gimid/warga#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by reopucino** [comment]
Hello there,

Just jump around here.. I'm Reza, daily as game programmer. Thanks to share the incredible platform.
